app/controllers/metrics_controller.rb

```diff
@@ -2,4 +2,23 @@ class MetricsController < ApplicationController
 	action_auth_level :index, :instructor
 	def index
 	end
+
+	action_auth_level :get_current_metrics, :instructor
+	def get_current_metrics
```

Comment:
this finally works, and I tried accessing the endpoint without logging in and it redirects me to the homepage as well